ajax/libs/handjs/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "handjs",
+  "version": "1.3.11",
+  "filename": "hand.min.js",
+  "description": "Hand.js is a polyfill for pointer events.",
+  "keywords": [
+    "pointer",
+    "touch",
+    "event"
+  ],
+  "license": "Apache-2.0",
+  "homepage": "http://github.com/deltakosh/handjs",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/deltakosh/handjs.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/deltakosht/handjs.git",
+    "basePath": "",
+    "files": [
+      "hand.js",
+      "hand.min.js"
+    ]
```

Comment:
I don't think so, if I don't make a mistake and regarding to the gulpfile, css and hand.base are source files used to build "hand.js" and "hand.min.js". I will talk to the author to know if he can refactor his repo to split src and dist files, if you want.